src/qt/veil.cpp

```diff
@@ -649,12 +649,15 @@ int main(int argc, char *argv[])
         return EXIT_FAILURE;
     }
 
-    // Preferences check
-    // If the UI settings have a different value than the server args then use the UI settings.
+    // Automint denom
+    // If nautomintdenom is set, use it
+    // Else use saved UI settings
     QSettings* settings = getSettings();
```

Comment:
Since settings isn't used anywhere else, this should be moved into the if not IsArgSet context, so it doesn't waste the cycles if it's not going to be used.

Reply:
Yes I was thinking the same thing but left it as is for simplicity's sake. Will update